.github/workflows/e2e.yaml

```diff
@@ -38,6 +39,10 @@ on:
         default: "1.28"
       cluster_name:
         type: string
+      enable_local_zones:
```

Comment:
If we have a suite name for this different set of testing, can we just check the suite name rather than having a separate parameter here?